core/Network/TLS/Extra/Cipher.hs

```diff
@@ -23,8 +23,8 @@ module Network.TLS.Extra.Cipher
     , cipher_AES256_SHA1
     , cipher_AES128_SHA256
     , cipher_AES256_SHA256
-    , cipher_AES128GCM_SHA256
-    , cipher_AES256GCM_SHA384
+    , cipher_RSA_AES128GCM_SHA256
+    , cipher_RSA_AES256GCM_SHA384
```

Comment:
Network.TLS has historically not provided a built-in default list of ciphersuites, and consequently applications have needed to make an explicit choice. Though some many may have loaded this module and used `ciphersuite_all`, others are likely to have built their own lists from individual ciphers.

I'm therefore relunctant to endorse renaming existing ciphers.  Yes, if we were starting from scratch, we would likely have done what this patch proposes, but at this point, I think that the TLS &le; 1.2 cipher names need to stay unchanged, even if that means that the TLS 1.3 ciphers have less _natural_ names.